Workbooks/UpdateCompliance/AlertsTargetBuild/Alerts for Target build.workbook

```diff
@@ -0,0 +1,366 @@
+            "delimiter": ",",
+            "query": "\r\nlet _SnapshotTime = datetime({_SnapshotTime});\r\nlet _TargetBuild = \"{_TargetBuild}\";\r\nUCUpdateAlert\r\n| where TimeGenerated == _SnapshotTime\r\n| where UpdateCategory == \"WindowsQualityUpdate\" and TargetBuild == _TargetBuild\r\n| summarize by AlertClassification",
+            "crossComponentResources": [
+              "{mappedWorkspace}"
+            ],
+            "value": [
+              "value::all"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "selectAllValue": "ALL",
+              "showDefault": false
+            },
+            "defaultValue": "value::all",
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "Filters"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let _SnapshotTime = datetime({_SnapshotTime});\r\nlet _TargetBuild = \"{_TargetBuild}\";\r\nlet _AlertClassification = \"{_AlertClassification}\";\r\nUCUpdateAlert\r\n| where TimeGenerated == _SnapshotTime\r\n| where TargetBuild == _TargetBuild\r\n| where AlertStatus == \"Active\" and UpdateCategory == \"WindowsQualityUpdate\"\r\n| where iff(_AlertClassification has \"ALL\", true, _AlertClassification has tostring(AlertClassification))\r\n| summarize Devices=count() by AlertSubtype, AlertType, AlertClassification, TargetBuild, ErrorCode\r\n| project-reorder AlertSubtype, AlertClassification, ErrorCode, AlertType, Devices",
+        "size": 0,
+        "exportedParameters": [
+          {
+            "fieldName": "AlertSubtype",
+            "parameterName": "_AlertSubtype"
+          },
+          {
+            "fieldName": "AlertClassification",
+            "parameterName": "_AlertClassification",
+            "parameterType": 1
+          },
+          {
+            "fieldName": "Devices",
+            "parameterName": "_DevicesCount",
+            "parameterType": 1
+          }
+        ],
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{mappedWorkspace}"
+        ],
+        "gridSettings": {
+          "formatters": [
+            {
+              "columnMatch": "AlertSubtype",
+              "formatter": 1,
+              "formatOptions": {
+                "linkTarget": "WorkbookTemplate",
+                "linkIsContextBlade": true,
+                "workbookContext": {
+                  "componentIdSource": "workbook",
+                  "resourceIdsSource": "workbook",
+                  "templateIdSource": "static",
+                  "templateId": "community-workbooks/UpdateCompliance/Error Codes for Alert Subtype",
```

Comment:
can't access your storage account at th emoment, but make sure this works

it might just be `community-workbooks/UpdateCompliance/ErrorCodes` ? but double check